### PR TITLE
update VocaForAllCell UI

### PR DESCRIPTION
**Pull request by Odyflame**
# Description 
update VocaForAllCell UI

<div>

</div>

### CODE REVIEW

File: Vocabulary/VocaForAll/VocaForAllCell.swift [hunk 1]
```diff
@@ -17,65 +17,71 @@ class VocaForAllCell: UITableViewCell {
...
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.clipsToBounds = true
+        view.layoutMargins = UIEdgeInsets(top: 4, left: 4, bottom: 4, right: 4)
+        return view
     }()
     lazy var titleLabel: UILabel = {
         let label = UILabel()
         label.translatesAutoresizingMaskIntoConstraints = false
-        label.textColor = .black
+        label.font = .BalsamiqSansBold(size: 30)
```
Comment (Haeuncs) [inline Vocabulary/VocaForAll/VocaForAllCell.swift:55]: Is this the correct font setting?

File: Vocabulary/VocaForAll/VocaForAllCell.swift [hunk 2]
```diff
@@ -17,65 +17,71 @@ class VocaForAllCell: UITableViewCell {
...
+       let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
     lazy var authorLabel: UILabel = {
         let label = UILabel()
         label.translatesAutoresizingMaskIntoConstraints = false
-        label.numberOfLines = 1
+        label.numberOfLines = 0
         label.textColor = .gray
```
Comment (Haeuncs) [inline Vocabulary/VocaForAll/VocaForAllCell.swift:69]: Is this the correct color?

File: Vocabulary/VocaForAll/VocaForAllCell.swift [hunk 3]
```diff
@@ -17,65 +17,71 @@ class VocaForAllCell: UITableViewCell {
...
         return label
     }()
+    lazy var authorImageView: UIImageView = {
+       let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.image = UIImage(named: "icPicture")
+        imageView.contentMode = .scaleAspectFit
+        imageView.clipsToBounds = true
+        imageView.layer.cornerRadius = Constant.imageRadius
```
Comment (Haeuncs) [inline Vocabulary/VocaForAll/VocaForAllCell.swift:78]: Is this the correct radius?

**Review by Haeuncs — CHANGES_REQUESTED** [review]

**Comment by Odyflame** [comment]
check this issue, i'll update these things.

<div>

</div>

**Pull request merged**